packages/cli/src/cli.ts

```diff
@@ -806,6 +806,11 @@ normal maps and ETC1S for other textures, for example.`.trim()),
 		+ ' faster, less noisy output, but lower quality per output bit).',
 		{validator: program.BOOLEAN}
 	)
+	.option(
```

Comment:
Could you add the same option to the `uastc` command below, as well? Thanks!

Reply:
done